src/components/PuzzleGrid/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from "react";
+import { Col, Row } from "antd";
+import puzzledImage from "./../../images/habitat.jpg";
+import shuffle from "./../memorygame/shuffle";
+import "./index.css";
+
+const NUM_OF_TILES = 6;
+// PuzzleGrid is the component that generates and includes the tiles of the puzzle
+// it generates 6 tiles ordered vertically on top of each other
+// each tile's background is 1/6 of the image. After the tiles are generated they are put
+// in an array and ordered randomly
+// a user can click on one tile to select and then click on another tile to swap their places
+// the user will swap the places of each tile to put it in its correct position
+// when all tiles are placed in their correct position the full image is displayed
+const PuzzleGrid = () => {
+  const [currentPuzzleState, setCurrentPuzzleState] = useState([]);
+  // selectedTile state shows if there is a tile selected and which one is selected if any
+  const [selectedTile, setSelectedTile] = useState(null);
+  // if unsolved it displays the tiles and when solved it displays the full image
+  const [isSolved, setIsSolved] = useState(false);
+
+  const tilesGenerator = () => {
+    const newTilesArray = [];
+    // for loop to generate tiles where tiles are puzzle pieces
+    for (let tileId = 0; tileId < NUM_OF_TILES; tileId++) {
+      let tileTopStyling = tileId * 100;
+      // value of selected is used for styling purposes where the selected tile should be highlighted
+      // value of top is used for fitting the image inside the tile
+      let newTile = { id: tileId, selected: false, top: tileTopStyling };
```

Comment:
> The way i'm doing this is i'm placing the image inside a div and making the image 6 times bigger than the div then displaying the part that i want to display of that image. I think what you are saying is I should create a new div inside the <Row> and then add the CSS to that div instead of add it to the img. please correct me if i misunderstood


yeah what i was saying is that you should try to use the normal flow of block ordering in css. to put it another way, when i see the "negative top" used to order 6 things on top of each other i immediately go like "that's a really strange way to achieve this." 

you don't need to use <Row> but if what i'm reading is correct, you want the image to be 6 times bigger than the parent div, i think you might want to do something like display: block on the img and make the parent div bigger. usually we want things to "make sense", so the child of a DOM element ebing 6 times bigger than its parent can be pretty weird